CHANGES.rst

```diff
@@ -15,8 +15,15 @@ Deprecated
 Features added
 --------------
 
+* #12975: Add the possibility to avoid rendering a trailing comma in Python and
+Javascript multi-line signatures, via
+:confval:`python_trailing_comma_in_multi_line_signatures` and
+:confval:`javascript_trailing_comma_in_multi_line_signatures`, respectively.
+
 Bugs fixed
 ----------
 
+* #12975: Avoid rendering a trailing comma in C/C++ multi-line signatures.
```

Comment:
```suggestion
* #12975: Avoid rendering a trailing comma in C and C++ multi-line signatures.
```

Reply:
The use of "C/C++" makes it sound like they are one thing, but they are two very different languages, and also domains in Sphinx.